cmake/GetDepthAI.cmake

```diff
@@ -29,3 +29,9 @@ else()
     set(DepthAI_INCLUDE_DIRS ${CMAKE_INSTALL_PREFIX}/include ${CMAKE_INSTALL_PREFIX}/include/depthai-shared ${CMAKE_INSTALL_PREFIX}/include/depthai-shared/3rdparty ${CMAKE_INSTALL_PREFIX}/include/depthai ${CMAKE_INSTALL_PREFIX}/lib/cmake/depthai/dependencies/include)
     set(DepthAI_LIBRARIES depthai-core;depthai-opencv)
 endif()
+
+add_custom_target(cleanup_depthai_spdlog
+                  COMMAND rm -rf ${CMAKE_INSTALL_PREFIX}/lib/cmake/depthai/dependencies/include/spdlog
+)
+
+add_dependencies(cleanup_depthai_spdlog DepthAI_ext)
```

Comment:
`DepthAI_ext` is not defined if it's already built. Got the error of `The dependency target "DepthAI_ext" of target "cleanup_depthai_spdlog" does not exist.`

Reply:
I will look into that

Reply:
just pushed a fix